lib/wahwah/tag.rb

```diff
@@ -3,29 +3,29 @@
 module WahWah
   class Tag
     INTEGER_ATTRIBUTES = %i[disc disc_total track track_total]
-    INSPECT_ATTRIBUTES = %i[title artist album albumartist composer track track_total genre year disc disc_total]
+    INSPECT_ATTRIBUTES = %i[
+      title
+      artist
+      album
+      albumartist
+      composer
+      comments
+      track
+      track_total
+      genre
+      year
+      disc
+      disc_total
+      lyrics
+      duration
+      bitrate
```

Comment:
I wanted all the attributes to be inspectable, as I felt they were useful to have at a glance.